### PR TITLE
[DPEDE-1546] Create transfer list e2e tests

### PR DESCRIPTION
**Pull request by emerson-pereira**
https://ctl.atlassian.net/browse/DPEDE-1546

### CODE REVIEW

**Comment by xft-dev** [comment]
I checked this PR and Modal tests are failing because some trigger buttons are missed.

image

**Review by xft-dev — APPROVED** [review]
Nice job! The tests pass, I just had to run "npm run build:umd" before, in order to see the modal buttons.

File: cypress/e2e/chi-vue/transfer-list.cy.js [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+  before(() => {
+    cy.visit('tests/chi-vue/transfer-list.html');
+  });
+
+  describe('Base', () => {
+    beforeEach(() => {
+      cy.get("[data-cy='transfer-list-base']").as('transfer-list');
+    });
+
+    it(`should have class ${TRANSFER_LIST_CLASSES.TRANSFER_LIST_ELEMENT}`, () => {
```
Comment (josecarlosgonzalezv) [inline cypress/e2e/chi-vue/transfer-list.cy.js:27]: Uppercase, check in every test

```suggestion
    it(`Should have class ${TRANSFER_LIST_CLASSES.TRANSFER_LIST_ELEMENT}`, () => {
```

**Review by josecarlosgonzalezv — CHANGES_REQUESTED** [review]
Good work. I would include tests for the wildcard and locked properties. 

Also test the different movements between columns and in the right column:

* items to left
* items to right
* items up
* items down
* all items to left
* all  items to right

**Review by josecarlosgonzalezv — APPROVED** [review]

**Review by jllr — APPROVED** [review]

**Pull request merged**